fortuna/src/casus/CasusContainer.js

```diff
@@ -1,15 +1,15 @@
 //@flow strict
 
 import * as React from 'react';
-import BlockBank from './BlockBank.js';
+import BlockBank from './blockBank/BlockBank.js';
 import CasusEditor from './CasusEditor.js';
 
 class CasusContainer extends React.Component<{||}> {
 
 	render(): React.Node {
 		const containerStyle = {
 			padding: 20,
-			margin: 20
+			backgroundColor: '#00ff00'
```

Comment:
Not a big deal, but do we want to be putting styling in the JavaScript files or keep them in our css files?

Reply:
It's probably good to be consistent and just do CSS files for everything. I'll change that before I push